app/models/location.rb

```diff
@@ -17,4 +17,12 @@ def sort_alpha
     # require "pry";binding.pry
     member.order(:primary_member)
   end
+
+  # def order(sql)
+  #   member.order(sql)
+  # end
+  def sort(params)
+    # require "pry";binding.pry
+    member.where("people_in_membership > ?", params)
+  end
```

Comment:
we should get rid of these other old methods. Nice use of feeding params into the method as an arg. I'm curious if we could do this a class method on members, instead of a instance method on parent. We should Ask Brian or Jaminson about that.